tools/Makefile

```diff
@@ -1,2 +1,5 @@
 default:
 	$(MAKE) -C bbbfancontrol
+
+	# uncomment once bbbsupervisor compiles including all dependencies
```

Comment:
Nit: Maybe we can prefix this comment with `TODO(Stadicus): `?

(It doesn't mean you have to do the work, just that you have the most context for questions about the future work described in the comment.)